core/java/iesi-core/src/main/java/io/metadew/iesi/script/execution/instruction/data/text/JsonPathTraversal.java

```diff
@@ -0,0 +1,49 @@
+package io.metadew.iesi.script.execution.instruction.data.text;
+
+import com.jayway.jsonpath.JsonPath;
+import com.jayway.jsonpath.PathNotFoundException;
+import io.metadew.iesi.script.execution.instruction.data.DataInstruction;
+import lombok.extern.log4j.Log4j2;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Log4j2
+public class JsonPathTraversal implements DataInstruction {
+
+    private static final String TEXT = "text";
+    private static final String JSON_PATH = "jsonPath";
+
+    private static final Pattern PATTERN = Pattern.compile("\\s*(?<" + TEXT + ">.+?)\\s*,\\s*\\.(?<" + JSON_PATH + ">.+)");
+
+    @Override
+    public String getKeyword() {
+        return "text.jsonpath";
+    }
+
+    @Override
+    public String generateOutput(String parameters) {
+        Matcher inputParameter = PATTERN.matcher(parameters);
+        String result = "";
```

Comment:
No need to define empty string there. Instantiate the variable during the JsonPath.read()

Reply:
I define it to omit the issue in the screenshot below. Also to return at least an empty string when an error might occur.

![image](https://user-images.githubusercontent.com/24315874/147085433-1b56e1bc-136d-431b-b29f-1805426ddab2.png)

Reply:
I'll change it to this method

![image](https://user-images.githubusercontent.com/24315874/147086360-f21a3e68-440a-4089-918a-ade898c473dd.png)